src/service/voteService.ts

```diff
@@ -0,0 +1,27 @@
+import { ClientException } from "../common/error/exceptions/customExceptions";
+import statusCode from "../constants/statusCode";
+import { voteRepository, withOptionRepository, worryWithRepository } from "../repository"
+import { CreateVoteDTO } from "../createWorryVoteDTO";
+
+const createWorryVote = async (createVoteDTO: CreateVoteDTO) => {
+
+    const worryWith = await worryWithRepository.findById(createVoteDTO.worryWithId);
+    console.log(worryWith);
+
+    const worryOption = await withOptionRepository.findById(createVoteDTO.optionId);
+
+    if (!worryWith) {
+        throw new ClientException("해당하는 아이디의 걱정글이 존재하지 않습니다");
+    }
```

Comment:
p4. worryWith 변수 할당 후 바로 검증 해주면 좋을 것 같아요. 
```suggestion
    const worryWith = await worryWithRepository.findById(createVoteDTO.worryWithId);
    if (!worryWith) {
        throw new ClientException("해당하는 아이디의 걱정글이 존재하지 않습니다");
    }
```